faq/faq.md

```diff
@@ -0,0 +1,351 @@
+
+# Licensing and compliance {#licensing}
+
+## Which licenses are compatible? {#compatible-licenses}
+
+## Is there a standard format for declaring copyright? {#standard-copyright}
+
+Copyright year copyright holder email
+
+## Which years do I include in the copyright statement? {#years-copyright}
+
+## Do I need to include both GPL-3.0-or-later and GPL-3.0-only in my repository? {#gpl-plus}
+
+Members of the GPL family are listed separately in the SPDX License List as
+-only and -or-later, even though the license texts are identical. If you have
+code under only one of these licenses, we recommend that you only include that
+one license.
+
+If you have code under both licenses, we recommend that you include both
+licenses separately.
+
+If you do not want to include both licenses separately (e.g., for space
+reasons), you can create the file `LICENSES/GPL-3.0.txt` and add the following
+header:
+
+```
+Valid-License-Identifier: GPL-3.0-only
+Valid-License-Identifier: GPL-3.0-or-later
+License-Text:
+
+[license text]
+```
+
+## What are license exceptions and what do I do with them? {#license-exceptions}
+
+## Do I use SPDX-Copyright, Copyright, or ©? {#copyright-symbol}
+
+## Which files are copyrightable? {#what-is-copyrightable}
+
+All files that are original works of authorship are copyrightable. In essence,
+if someone sat down typing their own original thoughts on a keyboard, then that
+person holds copyright over the output. Common examples are source code,
+documentation, audio, and video.
+
+There are some edge cases, however. For instance, the program `print("Hello,
+REUSE!")` probably does not meet the threshold of originality. Similarly, data
+files and configuration files may not meet that threshold either.
+
+## What to do with uncopyrightable files? {#uncopyrightable}
+
+There are two things that you can do with such a file to make sure that its
+copyright and licensing is recorded. The first option is to simply use your
+regular copyright and license header for this file. There is nothing that stops
+you from claiming copyright over your own works.
+
+The alternative is to waive your copyright by using the
+[CC0](https://creativecommons.org/publicdomain/zero/1.0/) license.
+
+It is important to note that you can only do this for your own works. If the
+file was authored by someone else, you must declare their copyright and license
+in the header.
+
+## How do I copy someone else's work? {#copy-work}
+
+If someone else has made their work available for you to use and copy, you can
+incorporate their work into your project. When you put the work in one of the
+files in your project, you should add an `SPDX-Copyright` tag for the copyright
+holder(s) and an `SPDX-License-Identifier` tag for the license(s) under which
+the work was made available.
+
+If the work was licensed differently from your project, you should verify
+whether the licenses are compatible, and add the new licenses to your project.
```

Comment:
```suggestion
whether the licenses are compatible, and add the new licenses to your project. See {#step-1} for how to add a more licenses to your project.
```

Reply:
I reckon this is a little bit extraneous.